src/Raven.Server/Documents/Handlers/Batches/Commands/MergedBatchCommand.cs

```diff
@@ -465,12 +469,20 @@ protected override long ExecuteCmd(DocumentsOperationContext context)
                     tpl.Reply[tpl.FieldName] = changeVector;
             }
         }
+
+        // We are requested to do not return result.
+        if (IncludeReply == false)
```

Comment:
Why does this matter? 
Under what conditions do we not want to return a result? Shouldn't we check that when we add items to the `Reply` ?

Reply:
This will essentially remove everything we use for tracking the changes at our clients. When we remove that we dont need to respond with etags and all those things... we can just execute blindly and return OK.

Reply:
So why _add_ to the `Reply` in the first place?

Reply:
Do not clear here, just avoid executing `Reply.Add`